### PR TITLE
RELATED: RAIL-2755 add hidden DashboardView examples, fix bugs

### PR DESCRIPTION
**Pull request by no23reason**
* fix widget alerts on anonymous Bear backends
* fix drill targets in InsightRenderer
* improve performance of ExampleWithSource component
* add a few DashbaordView examples in `/hidden/dashboard-view`

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: examples/sdk-examples/src/index.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import "@gooddata/sdk-ui-filters/styles/css/main.css";
 import "@gooddata/sdk-ui-charts/styles/css/main.css";
 import "@gooddata/sdk-ui-pivot/styles/css/main.css";
 import "@gooddata/sdk-ui-kit/styles/css/main.css";
+import "@gooddata/sdk-ui-ext/styles/internal/css/dashboard_embedding.css";
+import "@gooddata/sdk-ui-ext/styles/internal/css/layout.css";
```
Comment (kandl) [inline examples/sdk-examples/src/index.tsx:13]: This should not be necessary after we merge https://github.com/gooddata/gooddata-ui-sdk/pull/877 (layout.scss is imported in dashboard_embedding.scss)

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1839/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3447/ : SUCCESS in 14m 50s

File: examples/sdk-examples/src/index.tsx (hunk 1 above)
Comment (no23reason) [reply examples/sdk-examples/src/index.tsx]: Removed the line

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1840/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3449/ : SUCCESS in 14m 19s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/587/ : SUCCESS in 30s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/848/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3451/ : SUCCESS in 19m 34s

**Pull request merged**